### PR TITLE
fix(claiming-process): transform nanowits from claiming in wits

### LINKED ISSUE #1384: Handle claiming file tokens as nanowits
Hereinafter, the claiming file includes tokens as nanowits instead of wits. Sheikah should handle them according to that

### PR DESCRIPTION
**Pull request by Tommytrg**
Close #1384

### CODE REVIEW

**Comment by aesedepece** [comment]
My point is that everything not facing the user should be dealt as nanowits, and only convert to wits in the UI.

**Comment by aesedepece** [comment]
I think we can truncate to 2 decimal positions for the sake of legibility:
`4,102.72,257,625` vs. `4,102.72`

**Comment by aesedepece** [comment]
Also, numbers like `67556739.0241109` are being formated like `67,556,739.0,241,109`, which has wrong grouping on the decimal side.

**Comment by aesedepece** [comment]
This is still generating addresses of amount `50`, while the smallest possible value should now be `50_000_000_000`.

**Comment by aesedepece** [comment]
Specifically this line:
https://github.com/witnet/sheikah/blob/9e6588e22c729c1ce8dfab190cac9d3eac7c7b1f/src/utils.js#L484

should read:
```js
      return new Array(x).fill(CLAIMING_ADDRESS_MIN_NANOWITS * 10 ** exp)
```

**Review by aesedepece — APPROVED** [review]

File: src/background.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ ipcMain.on('shutdown-finished', () => {
 
 // Exit cleanly on request from parent process in development mode.
 if (isDevelopment) {
-  main()
+  // main()
```
Comment (tmpolaczyk) [inline src/background.js:126]: ?

**Review by gabaldon — APPROVED** [review]

**Pull request merged**